bindings/node/src/Version.cpp

```diff
@@ -0,0 +1,140 @@
+#include <arbiter/Version.h>
+#include "Version.h"
+
+namespace ArbiterNodeBindings {
+
+using v8::Context;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::FunctionTemplate;
+using v8::Isolate;
+using v8::Just;
+using v8::Local;
+using v8::Maybe;
+using v8::Nothing;
+using v8::Number;
+using v8::Object;
+using v8::Persistent;
+using v8::String;
+using v8::Value;
+
+Persistent<Function> SemanticVersion::constructor;
+
+SemanticVersion::SemanticVersion(unsigned major, unsigned minor, unsigned patch, Maybe<char *> prereleaseVersion, Maybe<char *> buildMetadata) {
+  _semanticVersion = ArbiterCreateSemanticVersion(
+    major,
+    minor,
+    patch,
+    (prereleaseVersion.IsJust() ? prereleaseVersion.FromJust() : NULL),
+    (buildMetadata.IsJust() ? buildMetadata.FromJust() : NULL)
+  );
+}
+
+SemanticVersion::SemanticVersion(const char *string) {
+  _semanticVersion = ArbiterCreateSemanticVersionFromString(string);
+}
+
+SemanticVersion::~SemanticVersion() {
+}
+
+void SemanticVersion::Init(Isolate* isolate) {
+  // Prepare constructor template
+  Local<FunctionTemplate> tpl = FunctionTemplate::New(isolate, New);
+  tpl->SetClassName(String::NewFromUtf8(isolate, "SemanticVersion"));
```

Comment:
Related: I've been going back and forth on whether it's worth providing ES6 classes that have property getters (think _computed properties_ in Swift).

That way things like `version.major` would work instead of `version.getMajorversion()`, but I'm not sure if the overhead outweighs the benefit:
- we'd need to subclass or create thin wrappers around everything exported from the native bindings, which may introduce more potential for bugs
- we may need to transpile to ES5 if wanting to target older versions of Node

That was the motiviation behind https://github.com/ArbiterLib/Arbiter/pull/68/commits/9a465941c054a6734c4b64f75f0ea5973233d252.

I think it could be a nicer API so will give it another shot once these bugs are worked out.